project/src/components/submit-comment-form/submit-comment-form.tsx

```diff
@@ -1,25 +1,48 @@
-// import { DetailedHTMLProps, InputHTMLAttributes } from 'react';
+import { useState } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import { postComment } from '../../store/api-actions';
+import { ThunkAppDispatch } from '../../types/action';
+import { CommentPost } from '../../types/comment-post';
 
 type SubmitCommentFormProps = {
-  handleRatingStarSelect: (value: string) => void,
-  handleCommentTextInput: (value: string) => void,
+  id: number,
 }
 
-function SubmitCommentForm({ handleRatingStarSelect, handleCommentTextInput }: SubmitCommentFormProps): JSX.Element {
-  const ratingInputChangeHandler = (evt: React.ChangeEvent<HTMLInputElement>) => {
-    handleRatingStarSelect(evt.target.value);
+const mapDispatchToProps = (dispatch: ThunkAppDispatch) => ({
+  onSubmit(id: number, comment: CommentPost) {
+    dispatch(postComment(id, comment));
+  },
+});
+
+const connector = connect(null, mapDispatchToProps);
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
+type ConnectedComponentProps = PropsFromRedux & SubmitCommentFormProps;
+
+function SubmitCommentForm({ id, onSubmit }: ConnectedComponentProps): JSX.Element {
+  const [commentStarValue, setCommentStarValue] = useState<string>('');
+  const [commentTextValue, setCommentTextValue] = useState<string>('');
+
+  const ratingInputChangeHandle = (evt: React.ChangeEvent<HTMLInputElement>) => {
+    setCommentStarValue(evt.target.value);
   };
 
-  const commentTextareaChangeHandler = (evt: { target: { value: string; }; }) => {
-    handleCommentTextInput(evt.target.value);
+  const commentTextareaChangeHandle = (evt: { target: { value: string; }; }) => {
```

Comment:
```suggestion
  const handleCommentChange = (evt: { target: { value: string; }; }) => {
```
используй нормальный тип для evt